### PR TITLE
Update System.Security.Cryptography.Xml

### PR DESCRIPTION
**Pull request by dkurepa**
https://github.com/dotnet/arcade-services/issues/2832

### CODE REVIEW

**Comment by dkurepa** [comment]
testing the change in https://dev.azure.com/dnceng/internal/_build/results?buildId=2243874&view=results

File: src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <PackageReference Include="Microsoft.Build" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
     <PackageReference Include="Microsoft.DotNet.VersionTools" />
+    <PackageReference Include="System.Security.Cryptography.Xml" />
```
Comment (premun) [inline src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj:19]: Please add a comment explaining we don't need this package but only pin it as it's a transitive dependency which is alerting in CG

Comment (dkurepa) [reply src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj:20]: Added. The pipeline run was also didn't have any CG issues

File: Directory.Packages.props [hunk 2]
```diff
@@ -116,6 +116,8 @@
     <PackageVersion Include="System.Text.Encodings.Web" Version="7.0.0" />
     <PackageVersion Include="System.ValueTuple" Version="4.5.0" />
     <PackageVersion Include="Verify.NUnit" Version="19.6" />
+    <!-- This package can be removed after we migrate to .net7 or higher. Was needed for https://dnceng.visualstudio.com/internal/_workitems/edit/5117 -->
```
Comment (premun) [inline Directory.Packages.props:119]:
```suggestion
    <!-- Pin of a transitive dependency. Can be removed after we migrate to net7.0+
         https://dnceng.visualstudio.com/internal/_workitems/edit/5117 -->
```

File: src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj [hunk 3]
```diff
@@ -16,6 +16,8 @@
     <PackageReference Include="Microsoft.Build" />
     <PackageReference Include="Microsoft.Build.Tasks.Core" />
     <PackageReference Include="Microsoft.DotNet.VersionTools" />
+    <!-- This package can be removed after we migrate to .net7 or higher. Was needed for https://dnceng.visualstudio.com/internal/_workitems/edit/5117 -->
```
Comment (premun) [inline src/Maestro/Microsoft.DotNet.Maestro.Tasks/Microsoft.DotNet.Maestro.Tasks.csproj:19]:
```suggestion
    <!-- Pin of a transitive dependency. Can be removed after we migrate to net7.0+
         https://dnceng.visualstudio.com/internal/_workitems/edit/5117 -->
```

**Review by premun — COMMENTED** [review]
I think the comments don't make it clear the package is not used by this project directly, so suggested a new comment

**Review by premun — APPROVED** [review]

**Pull request merged**